catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/FederationAdminServiceImpl.java

```diff
@@ -114,8 +122,12 @@
 
     private FilterBuilder filterBuilder;
 
+    private int pollerInterval = 0;
```

Comment:
Is it set to 0 by default to make sure when if it doesn't get injected properly it doesn't blow up?